src/frontend/src/pages/WorkPackageDetailPage/WorkPackageEditContainer/WorkPackageEditContainer.tsx

```diff
@@ -189,21 +199,33 @@ const WorkPackageEditContainer: React.FC<WorkPackageEditContainerProps> = ({ wor
         setManager={setManagerId}
       />
       <PageBlock title="Blocked By">
-        {blockedBy.map((_element, i) => {
-          return (
-            <Grid item sx={{ display: 'flex', alignItems: 'center', mb: 1 }}>
-              <TextField required autoComplete="off" {...register(`blockedBy.${i}.wbsNum`)} sx={{ width: 1 / 10 }} />
-              <IconButton type="button" onClick={() => removeBlocker(i)} sx={{ mx: 1, my: 0 }}>
-                <DeleteIcon />
-              </IconButton>
-            </Grid>
-          );
-        })}
-        <Button variant="contained" color="success" onClick={() => appendBlocker({ wbsNum: '' })} sx={{ mt: 2 }}>
-          + ADD NEW BLOCKER
-        </Button>
+        <FormControl fullWidth>
+          <Controller
+            name="blockedBy"
+            control={control}
+            render={({ field: { onChange, value: formValue } }) => (
+              <Autocomplete
+                isOptionEqualToValue={(option, value) => {
+                  console.log(option, value);
+                  return option.id === value.id;
+                }}
+                filterSelectedOptions
+                multiple
+                options={blockedByOptions}
+                getOptionLabel={(option) => option.label}
+                onChange={(_, value) => onChange(value.map((v) => v.id))}
+                value={formValue.map((v: string) => {
+                  let change = blockedByOptions.find((o) => o.id === v);
```

Comment:
no reason for this to be let, you can change it to const

Reply:
ah this was when i messing with the weird type stuff